docs/en/enterprise-edition/policy-reference/sast-policies/javascript-policies/sast-policy-196.adoc

```diff
@@ -0,0 +1,60 @@
+== DOM updates should not lead to cross-site scripting (XSS) attacks
```

Comment:
Options:

- Unsanitized DOM updates lead to cross-site scripting (XSS) attacks
- DOM updates lead to cross-site scripting (XSS) attacks